src/helpers/tools.js

```diff
@@ -8,7 +8,7 @@ import {
 } from 'path';
 import recursive from 'recursive-readdir-async';
 import {
-    exists, remove, ensureSymlink, chmod,
+    existsSync, remove, ensureSymlink, chmod,
```

Comment:
Wrong method is used. We should use `async` method. Vuems must work in async mode.  
`Exists` is deprecated but Node proposes to use 'fs.stat()' or `fs.access()` instead of `fs.exists()`.
Or use this https://github.com/jprichardson/node-fs-extra/blob/HEAD/docs/pathExists.md

Reply:
That doesn't make async mode at all. It's synchronously waiting for if statement and then it's doing next job.